src/installer/tests/HostActivation.Tests/FrameworkResolution/FrameworkResolution.cs

```diff
@@ -103,6 +103,51 @@ public void FrameworkResolutionError_ListOtherArchitectures()
             }
         }
 
+        [Theory]
+        [InlineData("6.1.0", "6.1.2", "6.1.3")] // Roll forward to 6.1.3 when 6.1.2 is disabled
+        [InlineData("6.1.0", "6.1.3", "6.1.2")] // Roll forward to 6.1.2 when 6.1.3 is disabled
+        [InlineData("6.1.3", "6.1.3", ResolvedFramework.NotFound)] // Fail when the only matching version is disabled
+        [InlineData("7.2.0", "7.2.3", ResolvedFramework.NotFound)] // Fail when the only matching version is disabled
+        public void DisabledVersions_Single(string requestedVersion, string disabledVersion, string expectedResolution)
```

Comment:
Should we also test the environment variable with "no content" and one with just whitespace?